src/views/list.rs

```diff
@@ -62,18 +65,21 @@ where
         id.update_state(ListUpdate::SelectionChanged);
     });
     let stack = v_stack_from_iter(iterator.into_iter().enumerate().map(move |(index, v)| {
-        Item {
-            data: ViewData::new(Id::next()),
-            selection,
-            index,
-            child: Box::new(v),
-        }
-        .on_click_stop(move |_| {
-            if selection.get_untracked() != Some(index) {
-                selection.set(Some(index));
-                id.update_state(ListUpdate::Accept);
+        container(
```

Comment:
This container should be removed. You can just apply the class directly to `Item`.